etc/adminhtml/system/transaction/creditcard/tds.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © 2017 Magento. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<include xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_include.xsd">
+    <group id="pagarme_pagarme_transaction_creditcard_3ds" translate="label comment" showInDefault="1" showInWebsite="1" showInStore="1" sortOrder="90">
+        <label>3DS</label>
+        <frontend_model>Pagarme\Pagarme\Block\Adminhtml\Form\Fieldset\CreditCardGatewayFieldset</frontend_model>
+        <field id="tds_active" showInDefault="1" showInStore="1" showInWebsite="1" sortOrder="20" translate="label" type="select">
+            <label>Active</label>
+            <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+            <config_path>payment/pagarme_creditcard/tds_active</config_path>
+        </field>
+        <field id="order_with_tds_refused" showInDefault="1" showInStore="1" showInWebsite="1" sortOrder="20" translate="label" type="select">
+            <label>Proceed with 3DS order declined?</label>
```

Comment:
Sugestão:
EN: Continue with an order that was rejected by 3DS?
PT: Continuar com um pedido que foi recusado pelo 3DS?

Reply:
Alterado.